polaris-react/build/esm/styles.css

```diff
@@ -5576,6 +5606,9 @@ li:first-of-type > .Polaris-Listbox-TextOption {
     border: var(--p-border-width-100) solid transparent;
   }
 }
+.Polaris-RadioButton__Backdrop {
+  position: relative;
+}
```

Comment:
The way SASS resolves mixins & nested rules is mostly the same as how `postcss-mixins` and `postcss-nesting` does it, except for the case where a mixin comes _after_ a nested rule; the final CSS output will be in a slightly different order ([see this explanation for
details](https://github.com/csstools/postcss-plugins/pull/1312)).

This is only an issue if any declarations following the mixin (which follows a nested rule) are used to overwrite declarations the mixin inserts.

Thankfully, there's an easy fix which requires moving the mixin call above any nested rules. We only had to do this twice (for `Navigation` & `ActionList`), the rest we left the source alone and hence the slighly changed output css above.